src/fourdvar/datadef/abstract/_physical_abstract_data.py

```diff
@@ -124,6 +124,7 @@ def unc(spc):
             "SDATE": np.int32(dt.replace_date("<YYYYDDD>", date_defn.start_date)),
             "EDATE": np.int32(dt.replace_date("<YYYYDDD>", date_defn.end_date)),
             "VAR-LIST": "".join([f"{s:<16}" for s in self.spcs]),
+            "openmethane_version": os.getenv("OPENMETHANE_VERSION", "development"),
```

Comment:
Using snake case for `openmethane_version` contrasts the existing attributes which are ALL-CAPS. However, in trying to follow CF Conventions for new attributes, I've adopted their style which is lowercase, with _ as a separator.

https://cfconventions.org/Data/cf-conventions/cf-conventions-1.11/cf-conventions.html#attribute-appendix